package.json

```
@@ -7,7 +7,8 @@
     "build": "npx prisma generate && next build",
     "start": "next start",
     "lint": "next lint",
-    "postinstall": "prisma generate"
+    "postinstall": "prisma generate",
+    "test": "npx jest --runInBand  landingPage.test.js"
```

Changing so that this command works for future tests as well

```suggestion
    "test": "npx jest --runInBand"
```